src/device-registry/controllers/create-activity.js

```diff
@@ -143,7 +143,6 @@ const activity = {
       }
 
       let request = Object.assign({}, req);
```

Comment:
thanks @BenjaminSsempala , for some reason, I cant see the removal of the `deviceName` thing from this PR? Do you have the latest `staging`?